src/core/dbus-cgroup.c

```diff
@@ -826,6 +829,31 @@ int bus_cgroup_set_property(
 
                 return 1;
 
+        } else if (STR_IN_SET(name, "MemoryLow", "MemoryHigh", "MemoryMax")) {
```

Comment:
`"\n"` in  `unit_write_drop_in_private_format(u, mode, name, "%s=max", name);` fixes that